website/static/js/signIn.js

```diff
@@ -15,10 +15,19 @@ var SignInViewModel = oop.extend(formViewModel.FormViewModel, {
     constructor: function () {
         var self = this;
         self.super.constructor.call(self);
-        self.username = ko.observable('').extend({
-            required: true,
-            email: true
-        });
+        var urlEmail = $osf.urlEmail();
```

Comment:
I see now; the `existing_user` param is used to prepopulate the input field. That said, `urlEmail` seems unnecessary. You should be able to read the parameter the regular way, as in

```js
var existingUserEmail = decodeURIComponent($osf.urlParams().existing_user);